src/main/java/frc/robot/commands/LevelChargingStationCommand.java

```diff
@@ -1,14 +1,17 @@
 package frc.robot.commands;
 
 import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.PIDCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants.LimelightDirections;
 import frc.robot.common.Odometry;
+import frc.robot.subsystems.LEDSubsystem;
 import frc.robot.subsystems.SwerveDriveSubsystem;
 
 public class LevelChargingStationCommand extends SequentialCommandGroup {
```

Comment:
Because this is a `SequentialCommandGroup`, it runs commands one at a time, in the same order as listed in `addCommands`. Since you want to be running the `balanceRobot` command *at the same time* as the LEDs change, you need to make the command a `ParallelCommandGroup`, which runs all the commands in `addCommands` at the same time.